tests/osaka/eip7939_count_leading_zeros/test_count_leading_zeros.py

```diff
@@ -276,3 +278,95 @@ def test_clz_jump_operation(
         post[callee_address] = Account(storage={"0x00": expected_clz})
 
     state_test(pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.valid_from("Osaka")
+@pytest.mark.parametrize("bits", [0, 16, 64, 128, 255])
+def test_clz_initcode_context(state_test: StateTestFiller, pre: Alloc, bits: int):
+    """Test CLZ opcode behavior when creating a contract."""
+    code = Op.SSTORE(1, Op.CLZ(1 << bits)) + Op.STOP
+
+    code_address = pre.deploy_contract(code=code)
+
+    init_code = Op.SSTORE(0, Op.CLZ(1 << bits))
+
+    creation_code = (
+        init_code
+        + Op.EXTCODECOPY(
+            address=code_address,
+            dest_offset=0,
+            offset=0,
+            size=Op.EXTCODESIZE(code_address),
+        )
+        + Op.RETURN(0, Op.MSIZE)
```

Comment:
We could simplify this part by simply returning `Op.RETURN(0, 1)`, since we are not really calling the deployed contract, it does not matter what ends up deployed.

With this we can also remove `code` and its deployment from the test.